src/AntiCSRF.php

```diff
@@ -94,17 +94,17 @@ public static function getTokenArray($lockto = null)
         }
 
         if (empty($lockto)) {
-            $lockto = isset($_SERVER['REQUEST_URI']) 
+            $lockto = isset($_SERVER['REQUEST_URI'])
                 ? $_SERVER['REQUEST_URI']
                 : '/';
         }
 
-        if (\preg_match('#/$#', $lockto)) {
```

Comment:
Sorry, I didn't answer your other question.

> are you interested restoring php-5.3 compatibility?

No, I'm interested in only supporting 5.5+ until it's EOL'd, then only 5.6+. If the PHP version is never going to receive a security patch again, I don't want to inadvertently encourage users to pick it up.